memory/sum_check.py

```diff
@@ -32,8 +33,13 @@ def setUp(self):
         self.iter = int(self.params.get('iterations', default='5'))
         self.memsize = int(self.params.get(
             'mem_size', default=memory.meminfo.MemFree.k * 0.9))
-        self.ddfile = os.path.join(self.workdir, 'ddfile')
-        if (disk.freespace(self.workdir) // 1024) < self.memsize:
+        base_dir = self.teststmpdir
+        if disk.freespace("/home") > disk.freespace(self.teststmpdir):
```

Comment:
Is it fine checking only against /home and no other mountpoint ?

Reply:
yes. Thought of iterating through the mountpoints but more often than not has points like `boot, sys, proc` etc. and ending up excluding most, hence sticking to `/` and `/home`

Reply:
ah i recall that i thought i had  the same question for other PR which has same logic, may we can have a check for `/home , /mnt and /media` as some time user mounted extra disk on those partitions and we can also handle this via yaml